src/test/resources/test_data/test_set_0_skipped_first_choice/test_set_0_skipped_first_choice_cvr.json

```diff
@@ -648,32 +673,59 @@
       "@id" : "contest-001",
       "@type" : "CVR.CandidateContest",
       "ContestSelection" : [ {
-        "@id" : "A",
-        "@type" : "CVR.ContestSelection"
+        "@id" : "cs-a",
+        "@type" : "CVR.ContestSelection",
+        "Code" : [ {
+          "@type" : "CVR.Code",
+          "OtherType" : "vendor-label",
+          "Type" : "other",
+          "Value" : "A"
+        } ]
       }, {
-        "@id" : "B",
-        "@type" : "CVR.ContestSelection"
+        "@id" : "cs-b",
+        "@type" : "CVR.ContestSelection",
+        "Code" : [ {
+          "@type" : "CVR.Code",
+          "OtherType" : "vendor-label",
+          "Type" : "other",
+          "Value" : "B"
+        } ]
       }, {
-        "@id" : "C",
-        "@type" : "CVR.ContestSelection"
+        "@id" : "cs-c",
+        "@type" : "CVR.ContestSelection",
+        "Code" : [ {
+          "@type" : "CVR.Code",
+          "OtherType" : "vendor-label",
+          "Type" : "other",
+          "Value" : "C"
+        } ]
       }, {
-        "@id" : "D",
-        "@type" : "CVR.ContestSelection"
+        "@id" : "cs-d",
+        "@type" : "CVR.ContestSelection",
+        "Code" : [ {
+          "@type" : "CVR.Code",
+          "OtherType" : "vendor-label",
+          "Type" : "other",
+          "Value" : "D"
+        } ]
       } ]
     } ],
-    "ElectionScopeId" : "gpu-001"
+    "ElectionScopeId" : "gpu-election"
   } ],
-  "GeneratedDate" : "2019-04-06T22:04:57-07:00",
+  "GeneratedDate" : "2019-04-22T21:55:39-07:00",
   "GpUnit" : [ {
-    "@id" : "gpu-001",
+    "@id" : "gpu-election",
     "@type" : "CVR.GpUnit",
-    "OtherType" : "election scope gpunit",
+    "Name" : "TestSets_Single_winner",
+    "OtherType" : "Election Scope Jurisdiction",
     "Type" : "other"
   } ],
   "ReportGeneratingDeviceIds" : [ "rd-001" ],
   "ReportingDevice" : [ {
     "@id" : "rd-001",
-    "@type" : "CVR.ReportingDevice"
+    "@type" : "CVR.ReportingDevice",
+    "Application" : "RCV Universal Tabulator",
```

Comment:
Just realized I've been calling it the "Universal RCV Tabulator". Recommend changing the instance of this in the code (in the window title).